dataprofiler/data_readers/avro_data.py

```diff
@@ -91,13 +91,19 @@ def is_match(
             options = dict()
 
         # get current position of stream
-        if data_utils.is_stream_buffer(file_path) and not isinstance(file_path, str):
+        if data_utils.is_stream_buffer(file_path):
```

Comment:
Because mypy doesn't detect type narrowing inside `data_utils.is_stream_buffer`, I figured casting like this would be a good compromise without having to add extra runtime checks.

Reply:
Oops I didn't leave the comment on the right line -- the relevant line is line 95.